doc/teacher.md

```diff
@@ -228,7 +235,7 @@ Comments can be left to every criterion checkpoint or to the whole group of grad
 
 It is also possible to mass approve submissions or mass release reviews for an assignment by choosing a minimal score and then pressing `Approve all` or `Release all` correspondingly. Every submission with a score equal or above the set minimal score will be approved or reviews to such submissions will be released.
 
-Grading criteria can be loaded from a file `criteria.json` in a corresponding assignment folder inside the `Tests` repository.
+Grading criteria will be loaded from a `criteria.json` file if it is added to the corresponding assignment folder inside the `Tests` repository.
```

Comment:
Prefer to use lower case repo names: `tests`.

Reply:
Fixed.